src/ftest.jl

```diff
@@ -85,29 +94,25 @@ function ftest(mods::LinPredModel...)
         pvals[i] = result.pval
     end
 
-    return CoefTable([SSR1s, SSR2s, df1s, df2s, MSR1s, MSR2s, fstats, pvals],
-    ["Model 1 SSR", "Model 2 SSR", "Model 1 df", "Model 2 df", "Model 1 MSR", "Model 2 MSR", "F statistic", "p-value"],
-    ["Model $(i-1):$i" for i in 2:nmodels])
+    return FTestTable(SSR1s, SSR2s, df1s, df2s, MSR1s, MSR2s, fstats, pvals)
```

Comment:
Showing the values for "Model 1" and "Model 2" isn't ideal in a table where there are more than two models. It would be better to have one line for each model, with a column giving the difference in SSR, df and MSR between the two. Do you see what kind of R output I'm taking as a reference here?